packages/venia-concept/src/components/SearchBar/searchBar.css

```diff
@@ -0,0 +1,72 @@
+.root {
+    background-color: rgb(var(--venia-grey));
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    transition-property: opacity, transform, height, padding;
```

Comment:
🙅 

The only properties we should be animating are the GPU-accelerated ones: `opacity` and `transform`. Other properties cause reflows and repaints when they change, which interfere with smooth animation. [Read more.](https://www.smashingmagazine.com/2016/12/gpu-animation-doing-it-right/)

If there's not an immediately clear solution, let's remove the transition for now and add it later. I don't want to hold this PR up because of an animation.

Reply:
Awesome, thanks for the info!

Reply:
I removed them. It does look a bit janky now but we can definitely revisit.